src/uint256.h

```diff
@@ -112,21 +111,11 @@ class uint256 : public base_blob<256> {
     static const uint256 ONE;
 };
 
-/* uint256 from const char *.
- * This is a separate function because the constructor uint256(const char*) can result
- * in dangerously catching uint256(0).
+/* uint256 from std::string_view.
+ * This is a separate function because the constructor uint256(std::string_view str) can result
+ * in dangerously catching uint256(0) via std::string_view(const char*).
```

Comment:
e9a436b7796cb285730d179fb3317ad50234ab04 changed it to: `This is not a uint256 constructor because of historical fears of uint256(0) resolving to a NULL string and crashing.`